dhis-2/dhis-services/dhis-service-tracker/src/test/java/org/hisp/dhis/tracker/report/TrackerBundleImportReportTest.java

```diff
@@ -263,7 +267,13 @@ private TrackerTimingsStats createTimingStats()
 
     private TrackerValidationReport createValidationReport()
     {
-        return new TrackerValidationReport();
+        TrackerValidationReport report = new TrackerValidationReport();
+        report.addError(
+            new TrackerErrorReport( "", TrackerErrorCode.E9999, TrackerType.EVENT, CodeGenerator.generateUid() ) );
```

Comment:
Just since I noticed this now, `TrackerErrorReport` appears like overuse of `*Report` naming as it judging from the arguments is modelling an individual error but not a report. Same for the warning below. 

At some point there might also be a discussion needed about prefixing every class with `Tracker` - does not make code very readable.

Reply:
Totally agree, I wrote about this in the PR description :) glad we are seeing this the same way :)